packages/gatsby-transformer-pdf/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "gatsby-transformer-pdf",
+  "description": "Gatsby transformer plugin for pdf",
+  "version": "2.0.0",
```

Comment:
Let's call this 1.0.0!

Reply:
gatsby-transform-pdf is already in version [1.0.8](https://www.npmjs.com/package/gatsby-transformer-pdf)

Reply:
I'm not completely sure, but it does not mean that the package must have a newer version?

Reply:
Heh, this is weird, I guess because that was stub package, maybe we should start with 1.0.9 then?

Reply:
Ok, i'll change it to version to 1.0.9